Felienne/hedy
# [BUG FIX] Tabs no longer overflow page layout

**Description**
In this PR we fix an issue where the increasing number of adventures would make the tabs overflow the code page. In this process also making the code editor think they could be wider than the page. We fix this issue by adding an `overflow-x-scroll` attribute to make sure that the overflow is caught by a scrollbar. This is not the nicest solution, but should work for now. We also improve some general styling of the tabs; for example, making sure the adventure title is always centered.

**Fix for**
This PR fixes #1778.

**How to test**
Navigate to any Hedy level and notice the changed UI of the adventure tabs. Navigate to either level 3 or level 5 in Dutch and notice that the tabs no longer overflow the code page.

**Screenshots**
![Schermafbeelding 2022-01-25 om 10 50 33](https://user-images.githubusercontent.com/38155424/150953806-89767495-14c2-4ed6-9c60-32a5cf0b3e05.png)
